src/panels/lovelace/cards/hui-weather-forecast-card.ts

```diff
@@ -72,7 +73,7 @@ class HuiWeatherForecastCard extends LitElement implements LovelaceCard {
 
   @state() private _subscribed?: Promise<() => void>;
 
-  private _sizeController = new ResizeController(this, {
+  private _heightSizeController = new ResizeController(this, {
```

Comment:
You should not add a second resize controller, you can use 1 for both height and width.

Reply:
Make the value something like: `{width: "...", height: "..."}`